runelite-client/src/test/java/net/runelite/client/plugins/xpglobes/XpGlobesTest.java

```diff
@@ -0,0 +1,138 @@
+package net.runelite.client.plugins.xpglobes;
```

Comment:
Add copyright header (see other files for reference)

Reply:
done